tests/views/resource_tests.py

```diff
@@ -354,3 +354,21 @@ def test_get_related_resource(self):
         ret = reference_resource.get_related_resources(user=user)
         relationship = ret["resource_relationships"][0]["relationshiptype_label"]
         self.assertEqual(relationship, en_preflabel)
+
+    def test_resource_report_good(self):
+        self.client.login(username="admin", password="admin")
+        url = reverse(
+            "resource_report", kwargs={"resourceid": self.resource_instance_id}
+        )
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
+    def test_resource_report_bad(self):
+        self.client.login(username="admin", password="admin")
+        resource = Resource.objects.get(resourceinstanceid=self.resource_instance_id)
+        resource.delete()
+        url = reverse(
+            "resource_report", kwargs={"resourceid": self.resource_instance_id}
+        )
+        response = self.client.get(url)
```

Comment:
This clutters the test output, unfortunately:
```py
Not Found: /report/f562c2fa-48d3-4798-a723-10209806c068
.
----------------------------------------------------------------------
Ran 1 test in 7.857s
```

The test in #11637 shows a way to silence this with assertLogs. I [asked](https://forum.djangoproject.com/t/best-patterns-for-handling-logging-of-4xx-and-5xx-status-codes-in-test-suite/35521) on the Django forum if there's a better way to do this with out removing all logging from tests, but I didn't get any bites.

Reply:
change in latest